jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/AnnotationUtil.java

```diff
@@ -0,0 +1,31 @@
+package org.eclipse.lsp4jakarta.jdt.core;
```

Comment:
Since this is a new file, I think we need a license on top. I see you have one in your test class, so you should be able to just copy that into here.

Reply:
Good catch, will add that !